react/AddProductBtn.tsx

```diff
@@ -81,14 +99,14 @@ class AddProductBtn extends Component<AddProductBtnProps, AddProductBtnState> {
     if (!isMobile) {
       showToast({
         action: {
-          label: intl.formatMessage({ id: 'wishlist-see-lists' }),
+          label: intl.formatMessage({ id: messages.seeLists.id }),
```

Comment:
```suggestion
          label: intl.formatMessage(messages.seeLists),
```

Reply:
And the same for the others.

Reply:
It doesn't work. Causes the following error: `Argument of type 'string' is not assignable to parameter of type 'MessageDescriptor'`